.github/workflows/deploy-production.yml

```diff
@@ -203,7 +202,7 @@ jobs:
         run: |
           terragrunt init -upgrade
           terragrunt validate
-          terragrunt plan -var "image=$(steps.get_ecr_tag.outputs.ecr_tag)" -out tfplan
+          terragrunt plan -var "image=${{ steps.get_ecr_tag.outputs.ecr_tag }}" -out tfplan
```

Comment:
I wish GHA had a supported linter 😞

Filed https://github.com/thunderbird/appointment/issues/471 to look into adding one in the future